kfs-purap/src/main/java/edu/arizona/kfs/module/purap/document/PurchaseOrderDocument.java

```diff
@@ -0,0 +1,68 @@
+package edu.arizona.kfs.module.purap.document;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kfs.sys.businessobject.Building;
+import org.kuali.rice.krad.util.ObjectUtils;
+import edu.arizona.kfs.sys.businessobject.BuildingExtension;
+
+/**
+ * Purchase Order Document
+ */
+public class PurchaseOrderDocument extends org.kuali.kfs.module.purap.document.PurchaseOrderDocument {
+	private static final org.apache.log4j.Logger LOG = org.apache.log4j.Logger.getLogger(PurchaseOrderDocument.class);
+
+    protected String routeCode;
+    protected Building buildingObj;
+
+    /**
+     * Gets the routeCodeObj attribute. 
```

Comment:
After reviewing PurchaseOrderDocument class code in KFS 6.0, I think we can simplify this getter/setter pair quite a bit. Please code the buildingObj getter/setter methods as follows. Please also remove inaccurate method comments.

    public Building getBuildingObj() {
   	if (ObjectUtils.isNull(buildingObj)) {
    		this.refreshReferenceObject("buildingObj");
    	}   	
                  return buildingObj;
    }
 
    public void setBuildingObj(Building buildingObj) {
    	this.buildingObj = buildingObj;        
    }